src/binary-reader-objdump.cc

```diff
@@ -195,6 +196,12 @@ void BinaryReaderObjdumpBase::PrintRelocation(const Reloc& reloc,
   printf("\n");
 }
 
+Offset BinaryReaderObjdumpBase::GetPrintOffset(Offset offset) const {
+  return
+      options_->section_offsets ?
+      offset - GetSectionStart(BinarySection::Code) : offset;
+}
```

Comment:
I find the multi-line ternery expression s less easy to read.  How about:

```
if (options_->section_offsets) {
   return offset - GetSectionStart(BinarySection::Code);
}
return offset;
```

Reply:
oops, I meant to let clang-format fix this before uploading. here's what it comes up with:
```
  return options_->section_offsets
             ? offset - GetSectionStart(BinarySection::Code)
             : offset;
```

Reply:
That looks better than the 2-line version, but I can do the if/else version if you prefer too

Reply:
I guess that is nicer yes.   lgtm